R/Model-validity.R

```diff
@@ -49,19 +49,19 @@ v_model_logistic_kadane <- function(object) {
 v_model_logistic_kadane_beta_gamma <- function(object) { # nolintr
   v <- Validate()
   v$check(
-    test_number(object@alpha, lower = .Machine$double.xmin, finite = TRUE),
+    is.scalar(object@alpha) & is.numeric(object@alpha) && object@alpha > 0,
     "Beta distribution shape parameter alpha must be a positive scalar"
```

Comment:
Hmm, why do we have this change here? I think `test_number` is better than `is.scalar and etc.` I guess this results from some merging issues.